pantheon-decoupled-example.php

```diff
@@ -71,10 +72,10 @@ function pantheon_decoupled_example_menu() {
  * Activate the plugin.
  */
 function pantheon_decoupled_example_activate() {
-	if ( !get_transient('pantheon_decoupled_example_created') ) {
+	if ( !get_transient('pantheon_decoupled_example_created') && !post_exists("Example Post with Image") ) {
```

Comment:
I'd love to figure out the root cause of this at some point, but this should do the trick for now.